makefile

```diff
@@ -144,6 +132,28 @@ else
    $(call mfem-info, NOT including $(CONFIG_MK))
 endif
 
```

Comment:
Had to move this code block to accommodate changing the miniapps list, based on whether code coverage was enabled.  Currently removing 'performance' miniapp due to very long run-time required with a debug build.  (Causes job to exceed time on travis )